modules/electrophysiology_browser/php/electrophysiologybrowserrowprovisioner.class.inc

```diff
@@ -47,10 +47,17 @@ class ElectrophysiologyBrowserRowProvisioner
               s.Visit_label                             AS Visit_Label,
               MIN(pf.AcquisitionTime)                   AS Acquisition_Time,
               MIN(pf.InsertTime)                        AS Insert_Time,
-              GROUP_CONCAT(DISTINCT pot.OutputTypeName) AS Links,
```

Comment:
The GROUP_CONCAT/DISTINCT combo prevents duplicates entries when a session contains multiple files of the same type (ex: 2 raw files will display only one raw link in the data table). For 2 files of mixed type, it will display raw twice in the links in the data table (since ''raw", and ''raw,derivative" are distinct).

Reply:
This issue has been resolved in the latest commit, and the picture in the PR description has been updated to show the new view!